py3status/formatter.py

```diff
@@ -345,8 +345,11 @@ def _check_valid_condition(self, get_params):
             variable = None
         value = self.value
 
+        # reject the condition?
+        if variable is None:
+            return not self.default
         # convert the value to a correct type
-        if isinstance(variable, bool):
+        elif isinstance(variable, bool):
```

Comment:
Changing the `if` to `elif` does not make sense here as the proceeding `if`feels unrelated.  Also as we cannot get here if the proceeding condition is met.  So this just makes this PR harder to review.

Reply:
I break them up. :+1: